tests/unit/test_map.py

```diff
@@ -75,3 +75,46 @@ def test_map_empty_geometry_column():
         "entry-date,geometry,legislation,name,notes,organisation,point,prefix,"
         "reference,start-date\r\n,,,,,,MULTIPOLYGON(),,,,,,,,\r\n"
     )
+
```

Comment:
this is testing the Map class so it should be in:
`tests/unit/phase/map.py`
also I can;t really to the method/function that this is testing from the name. Also the TestPipeline class may not be needed see [here](https://github.com/digital-land/digital-land-python/blob/main/tests/unit/phase/test_organisation.py) you can just feed in a dict pretty much and it's easier to read

Reply:
I think I've already commented on this, but there is already a unit/test_map class.
I put the code in there to be consistent with what has been done previously.
Would it not be confusing to have Map class tests in 2 locations.
If we move this one, then we have to move many others, too.

Reply:
I've changed the function names.
hope they're a bit more understandable.